apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImport.tsx

```diff
@@ -100,10 +97,7 @@ const SpreadsheetImport = ({
     }
 
     if (!file || !includes(UPLOAD_FILE_TYPES, file?.type) || !acceptedFileExtension(file)) {
-      ui.setNotification({
-        category: 'info',
-        message: 'Sorry! We only accept CSV or TSV file types, please upload another file.',
-      })
+      toast('Sorry! We only accept CSV or TSV file types, please upload another file.')
```

Comment:
Feel like this maybe should be an error?

Reply:
fair point actually haha